core/src/test/java/com/linecorp/armeria/internal/common/stream/ByteBufsInputStreamTest.java

```diff
@@ -96,6 +98,43 @@ void testPreBuffered() {
         assertThat(resultStrings.get(4)).isEqualTo("fifth");
     }
 
+    @Test
+    void testAdd_ignoreEmptyByteBuf() {
```

Comment:
https://github.com/line/armeria/blob/7561be45824b691a32e486d8d8a98c3ad930c32b/core/src/main/java/com/linecorp/armeria/internal/common/stream/ByteBufsInputStream.java#L208-L211

(FYI) @minwoox , thanks to above code, this test anyway passed without my change.
But anyway, I address your comment! (ignore empty bytebuf on add())